data/faq/steal_project_idea.md

```diff
@@ -0,0 +1,6 @@
+The whole point of 'Open Source Software' is to encourage "stealing".
+If you do honor original authors, licenses, and the "creator" of an
+original idea, there is no shame in saying "he had a great idea, I think
+I can implement it". It's called "collaboration". On the plus side, we will
+always take into account who came up with something and if you help
+others succeed, we'll definitely credit you for that.
```

Comment:
I'd add another paragraph explaining that we essentially want to work with people who can work together, not with "lone hackers".